### DISCUSSION TITLE
What caused this huge error(s)

### DISCUSSION THREAD

**Discussion by githubUser01946** [vladmandic/sdnext#1983] [Q&A]
I have had to paste it onto pastebin as it is MASSIVE

https://pastebin.com/2ubgJjNM

I was loading up the fork after doing a git pull.

SDXL 1.0 base is the selected model.

**Comment by brknsoul** [comment]
> SDXL 1.0 base is the selected model.

`15:15:48-992229 INFO     Pipeline: Backend.ORIGINAL`

SDXL requires the Diffusers backend.

**Comment by vladmandic** [comment]
as answered.

**Comment by githubUser01946** [comment]
How do I select this? 

Its working fine in A1111.

**Reply by vladmandic** [reply]
sdxl implementation is totally different, so "its working fine in a1111" doesn't mean anything.
step-by-step guide is in the wiki.
